lib/daru.rb

```diff
@@ -96,6 +96,8 @@ def plotting_library= lib
 require 'erb'
 require 'date'
 
+require 'open-uri'
```

Comment:
Better to require it where it is really needed (in io). This way the separation of concerns would be clearer.